src/pcm.rs

```diff
@@ -178,6 +178,23 @@ impl PCM {
         acheck!(snd_pcm_status(self.0, z.ptr())).map(|_| z)
     }
 
+    pub fn status_with_audio_htstamp_config(
+        &self,
```

Comment:
Functions that do not map 1-to-1 to an alsa-lib function should have a comment explaining what they do.

Reply:
Will do. Thanks for the helpful feedback.